Contest_API.md

```diff
@@ -2262,6 +2262,8 @@ By passing `group_id` with a valid group ID a scoreboard can be requested for th
 
 `contests/<id>/scoreboard?group_id=site1`
 
+If no `group_id` is given, the group specified by `deafault_scoreboard_group` in the contest endpoint is used, or all teams if it is `null`. 
```

Comment:
Typo in property name (and update if we change the name above).

I might be overly picky, but the current setup is that group_id is an optional parameter - you only add it when you want a specific group. To me this change makes it sound more like the user is expected to pass the group_id and we're specifying a default behaviour for when they don't. I'm ok if we want to include a mention of the default group id in the Scoreboard section, but I'd do that in the main part. The section on group_id shouldn't need to change b/c picking the same group as the default is maybe unnecessary, but isn't special in any way.